src/main/java/uk/gov/hmcts/reform/waworkflowapi/controllers/startworkflow/CreateTaskController.java

```diff
@@ -94,32 +94,32 @@ public CreateTaskController(EvaluateDmnService evaluateDmnService,
     }
 
     @PostMapping(path = "/workflow/message")
-    @ApiOperation("Sends a message to the underlying business process engine.")
+    @Operation(summary = "Sends a message to the underlying business process engine.")
     @ApiResponses({
```

Comment:
This uses ``@ApiResponses` but `evaluateDmn` doesn't ? Consistency ?

Reply:
good point